bouncer/shared/send_btc.ts

```diff
@@ -82,15 +86,31 @@ export async function sendBtc(
   confirmations = 1,
 ): Promise<string> {
   // Btc client has a limit on the number of concurrent requests
-  const txid = (await btcClientMutex.runExclusive(async () =>
-    btcClient.sendToAddress(address, amount, '', '', false, true, null, 'unset', null, 1),
-  )) as string;
-
-  if (confirmations > 0) {
-    await waitForBtcTransaction(logger, txid, confirmations);
+  let txid: string;
+  let attempts = 0;
+  const maxAttempts = 3;
+
+  while (attempts < maxAttempts) {
+    try {
+      txid = (await btcClientMutex.runExclusive(async () =>
+        btcClient.sendToAddress(address, amount, '', '', false, true, null, 'unset', null, 1),
+      )) as string;
+
+      if (confirmations > 0) {
+        await waitForBtcTransaction(logger, txid, confirmations);
+      }
+      return txid;
+    } catch (error) {
+      attempts++;
+      logger.warn(`Error sending BTC transaction (attempt ${attempts}): ${error}`);
+      if (attempts >= maxAttempts) {
+        throw error;
+      }
+      await sleep(1000);
+    }
   }
 
-  return txid;
+  return '';
```

Comment:
Would we not want to throw an error in this case instead of returning the empty string?